flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/cli/CliClientITCase.java

```diff
@@ -76,8 +82,8 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 /** Test that runs every {@code xx.q} file in "resources/sql/" path as a test. */
-@RunWith(Parameterized.class)
-public class CliClientITCase extends AbstractTestBase {
+@ExtendWith(TestLoggerExtension.class)
+public class CliClientITCase {
```

Comment:
```suggestion
class CliClientITCase {
```
It could be package private